### DISCUSSION TITLE
After the web is started, the check port is not up

### DISCUSSION THREAD

**Discussion by MentosL** [apache/skywalking#10822] [Q&A]
When I used the script to start the web service, it told me to start it normally, and then I accessed it through the form of ip+port, but the access failed, and then I checked the netstat port occupancy through the server, and I found that the port was not in use. At the same time, it can also be found by checking the java process through jps. What is the reason?

Environment: ubuntu 20.X
Serving: v9.4.0
es: 7.6.1

**Comment by MentosL** [comment]
![image](https://github.com/apache/skywalking/assets/33775732/0cbbd25a-5367-401e-a589-434df7b69e4d)

**Comment by Superskyyy** [comment]
Hi! @MentosL You may want to check the oap + ui logs, it will indicate why it fails.

**Reply by MentosL** [reply]
> Hi! @MentosL You may want to check the oap + ui logs, it will indicate why it fails.

Thank you replay. i change approach to run skywalking (docker),it's normal now, thank you
